config/data_sync_list/common.json

```diff
@@ -28,6 +28,12 @@
             "SyncDirection": "Active2Passive",
             "SyncType": "Immediate",
             "ExcludeFilesList": ["/var/lib/phosphor-fan-presence/control/"]
+        },
+        {
+            "Path": "/etc/credstore.encrypted/",
```

Comment:
Looks good to me but, I would like to confirm whether the credstore.encrypted could be common or IBM specific. @joseph-reynolds

I assume we could sync it i.e its not specific to the BMC...